### PR TITLE
Issue 17 - Utility 4 - Add awesomebar highlighting

### PR DESCRIPTION
**Pull request by FrancescoSTL**
This PR adds the switch case for the 'try it now' button on the awesome-bar utility sidebar 4.

Issue ref: #17

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -241,6 +241,9 @@ function showSidebar(sidebarProps, contentURL) {
                     case 'privateBrowsing':
                         highLight('privateWindow');
                         break;
+                    case 'highLightURL':
```
Comment (schalkneethling) [inline index.js]: I am going to be a total ass and say, let's make this highlightURL?

Comment (FrancescoSTL) [inline index.js]: Wow, *facepalm*. I didn't realize I capitalized the L haha

Comment (FrancescoSTL) [inline index.js]: I'll change the function capitalization in a separate PR I suppose.

**Comment by FrancescoSTL** [comment]
👍 r?

**Comment by schalkneethling** [comment]
r+ 🌈

**Pull request merged**